build.sbt

```diff
@@ -251,3 +252,25 @@ lazy val parboiledCore = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     Compile / sourceGenerators += generateActionOps.taskValue
   )
   .nativeSettings(nativeSettings)
+
+ThisBuild / githubWorkflowTargetTags ++= Seq("v*")
+ThisBuild / githubWorkflowPublishTargetBranches := Seq()
+
+ThisBuild / githubWorkflowJavaVersions := List(JavaSpec.temurin("8"), JavaSpec.temurin("11"), JavaSpec.temurin("17"))
```

Comment:
One important thing to note here is that the first value in this list is whats used for publishing (if we decide to do it). I am not sure what JVM parboiled2 is published under, but if its not JDK 8 I need to update this.

Reply:
JDK 8 should be fine at this time. Thank you!